internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -152,6 +152,19 @@ func resourceKubernetesClusterNodePoolSchema() map[string]*pluginsdk.Schema {
 			ForceNew: true,
 		},
 
+		"gpu_instance_profile": {
```

Comment:
Profile is redundant here, can we rename this to 
```suggestion
		"gpu_instance": {
```

Reply:
Got it! I've updated this PR.